package.json

```diff
@@ -44,18 +44,21 @@
     ]
   },
   "dependencies": {
-    "bloomfilter": "^0.0.18"
+    "bloomfilter": "^0.0.18",
+    "mime-types": "^2.1.22"
   },
   "devDependencies": {
     "@types/get-port": "^4.0.0",
+    "@types/mime-types": "^2.1.0",
     "@types/node": "^10.0.1",
     "@types/semver": "^5.5.0",
-    "ava": "^1.0.0",
+    "ava": "^1.3.1",
     "codecov": "^3.0.1",
     "get-port": "^4.0.0",
     "gts": "^0.9.0",
     "js-green-licenses": "^0.5.0",
-    "nyc": "^13.0.0",
+    "nyc": "^13.3.0",
+    "requirejs": "^2.3.6",
```

Comment:
Is this needed?

Reply:
I have no idea how did requirejs got into this, unless it has been added by `npm audit fix --force`. I am removing it anyway